docs/docs/deployment/code-locations/project-workspace-code-location.md

```diff
@@ -0,0 +1,137 @@
+---
+title: 'Comparing projects vs workspaces vs code locations'
+description: 'Understanding the differences and relationship between the projects, workspaces and code locations'
+sidebar_position: 50
+---
+
+Dagster uses several related terms to describe how your code is organized and deployed. While they may sound similar, each has a distinct meaning and purpose in the lifecycle of a Dagster deployment.
+
+## Project
+
+A **project** follows the general Python convention, a structured collection of source files, libraries, and configuration used for a specific application. The recommended Python project layout (centered around a `pyproject.toml`) looks like this:
+
+```
```

Comment:
I would update this diagram to reflect Dagster project structure (rename `my_package` to `my_project`, add definitions.py)